### PR TITLE
Add Warn Before Closing Dialog Demo

### PR DESCRIPTION
**Pull request by luiemilio**

### CODE REVIEW

File: how-to/use-platform/warn-before-closing-dialog/public/config/manifest.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
+{
```
Comment (johnman) [inline how-to/use-platform/warn-before-closing-dialog/public/config/manifest.json:1]: Can this be renamed to manifest.fin.json and get moved to the root of the public folder to get consistent with workspace starter and container?

File: how-to/use-platform/warn-before-closing-dialog/public/html/dialog.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="description" content="">
+    <title>View(s) preventing closing</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script type="module" src="../js/dialog.bundle.js"></script>
+    <style>
```
Comment (johnman) [inline how-to/use-platform/warn-before-closing-dialog/public/html/dialog.html:11]: We now have a common styleguide for starters shared across workspace starter and container starter. Is it possible to use it? E.g. https://github.com/built-on-openfin/container-starter/blob/264e139bd1f653591481848d4137f9ec6f154220/how-to/create-window/public/html/window.html#L8

File: how-to/use-platform/warn-before-closing-dialog/public/html/provider.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Provider</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (johnman) [inline how-to/use-platform/warn-before-closing-dialog/public/html/provider.html:9]: We now have a common styleguide for starters shared across workspace starter and container starter. Is it possible to use it? E.g. https://github.com/built-on-openfin/container-starter/blob/264e139bd1f653591481848d4137f9ec6f154220/how-to/create-window/public/html/window.html#L8

File: how-to/use-platform/warn-before-closing-dialog/public/html/view.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>View</title>
+    <script type="module" src="../js/view.bundle.js"></script>
+    <style>
```
Comment (johnman) [inline how-to/use-platform/warn-before-closing-dialog/public/html/view.html:10]: We now have a common styleguide for starters shared across workspace starter and container starter. Is it possible to use it? E.g. https://github.com/built-on-openfin/container-starter/blob/264e139bd1f653591481848d4137f9ec6f154220/how-to/create-window/public/html/window.html#L8

File: how-to/use-platform/warn-before-closing-dialog/public/html/window.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Warn Before Closing1</title>
+    <script type="module" src="../js/window.bundle.js"></script>
```
Comment (johnman) [inline how-to/use-platform/warn-before-closing-dialog/public/html/window.html:9]: We now have a common styleguide for starters shared across workspace starter and container starter. Is it possible to use it? E.g. https://github.com/built-on-openfin/container-starter/blob/264e139bd1f653591481848d4137f9ec6f154220/how-to/create-window/public/html/window.html#L8

**Review by johnman — CHANGES_REQUESTED** [review]
Hi, thanks for the contributions. Just added a few points to bring it into line with workspace and container starter samples.

File: how-to/use-platform/warn-before-closing-dialog/public/html/view.html [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    <link rel="stylesheet" href="/common/style/app.css" />
+</head>
+
+<body class="col fill gap20">
+    <div id="main-div" class="col fill gap20">
+        <h1 id="view-title">View</h1>
+        <p>
+            Type in the field below and try close the View or Window. A dialog will pop up indicating that this view may have unsaved changes.
+        </p>
+        <div id="input-div">
```
Comment (obany) [inline how-to/use-platform/warn-before-closing-dialog/public/html/view.html:19]: Could we use the standard approach for laying out forms, the style.html in common has an example of using fieldsets and labels

**Review by johnman — APPROVED** [review]

**Comment by adam-saland** [comment]
Need to update to use common fin, will add and merge.

**Comment by johnman** [comment]
@luiemilio hi Luis. When you have a spare cycle could you re-review this pull request? It would be good to get it in.

**Comment by adam-saland** [comment]
I think I did this in vnext... may have forgotten to merge

**Comment by adam-saland** [comment]
resolved in other PR/Branch

**Pull request closed without merging**